homeassistant/components/recorder/util.py

```diff
@@ -645,7 +645,7 @@ def wrapper(instance: _RecorderT, *args: _P.args, **kwargs: _P.kwargs) -> bool:
                 return job(instance, *args, **kwargs)
             except OperationalError as err:
                 if _is_retryable_error(instance, err):
-                    assert isinstance(err.orig, BaseException)
```

Comment:
Here I added the `noqa` because this assert is not about tests and so `pytest.raises` would not work.